src/main/java/rs/hybridit/model/Librarian.java

```diff
@@ -0,0 +1,22 @@
+package rs.hybridit.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import lombok.NoArgsConstructor;
+import rs.hybridit.dto.UserDto;
+
+@Entity
+@NoArgsConstructor
+public class Librarian extends User {
+
+	public Librarian(UserDto userDto) {
+		super(userDto);
+		List<Authority> authorities = new ArrayList<>();
+		Authority a = new Authority();
```

Comment:
You could extract this logic into User, something like setRole(...) and in implementation create the authorities list and assign authority with role